eks/aws-node-termination-handler/Settings.dhall

```diff
@@ -0,0 +1,100 @@
+let Prelude = (../../imports.dhall).Prelude
+
+let Pod =
+      let Image =
+            { Type = { repository : Text, tag : Text, pullPolicy : Text }
+            , default =
+                { repository = "amazon/aws-node-termination-handler"
+                , tag = "v1.2.0"
+                , pullPolicy = "IfNotPresent"
+                }
+            }
+      
+      let Resources =
```

Comment:
We need to make sure its not get stuck on `Pending` once deployed to production.